stdlib/src/Luna/Prim/Foreign.hs

```diff
@@ -0,0 +1,355 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Can we get this file wrapped to 80 characters?

Reply:
Done